src/ReleaseHistory.md

```diff
@@ -1,5 +1,8 @@
 # Sarif Driver and SDK Release History
 
+## **v2.1.17** [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.17) | [SDK](https://www.nuget.org/packages/Sarif.Sdk/2.1.17)
```

Comment:
>## [](start = 0, length = 2)

Heads up: I have a PR out that reverses the order of the release history, with newest versions first, as you naturally did here. If my PR goes in first, you'll have a merge to do (taking my version of the file in its entirety and prepending your change).

Reply:
Ok. I'll look for merge problems.